pkg/grep.go

```diff
@@ -0,0 +1,89 @@
+package pkg
```

Comment:
I know this is kind of a simple tool, but would it make sense to put grep to its own package? I don't know if there's need for this elsewhere, but just something that I was thinking. I'm also fine with it as-is.

Reply:
I also thought about it. I don't mind either way. I prefer extracting packages down the road as the need emerges - this leads to better design IMO. Feel free to extract it though when you do some work around that.